src/main.ts

```diff
@@ -850,6 +856,12 @@ export class Purchases {
       };
 
       const onError = (e: PurchaseFlowError) => {
+        const sessionEndErroredEvent = createCheckoutSessionEndErroredEvent({
+          errorCode: e.errorCode?.toString() ?? null,
+          errorMessage: e.message,
+          mode: "express_purchase_button",
+        });
+        this.eventsTracker.trackSDKEvent(sessionEndErroredEvent);
```

Comment:
If an error occurs during the initialization, we will send a `CheckoutSessionEndErroredEvent` without a `CheckoutSessionStartEvent` being sent. This seems incorrect to me.

<img width="741" height="272" alt="Image" src="https://github.com/user-attachments/assets/f66a1879-520f-4086-bee7-35c6cb642c15" />

Reply:
I can convert this one into a `createCheckoutFlowErrorEvent` or a `export function createCheckoutPaymentFormErrorEvent`

Will do in the next commit

Reply:
I moved all the events management in the component.
Reusing the handleError function.

In this way we should always have the simple createCheckoutFlowErrorEvent using the error code and message from the exception that was raised.